utils/environment.python.sh

```diff
@@ -2,9 +2,7 @@
 
 BASE_DIR=${FPGA_TOOL_PERF_BASE_DIR}
 # FIXME: fasm should be installed into the running Python environment.
-# FIXME: enable SYMBIFLOW when installed toolchain is available
-#export PYTHONPATH="${SYMBIFLOW}/python:$BASE_DIR:$BASE_DIR/third_party/fasm:$BASE_DIR/third_party/edalize:$PYTHONPATH"
-export PYTHONPATH="$BASE_DIR:$BASE_DIR/third_party/fasm:$BASE_DIR/third_party/edalize:$PYTHONPATH"
+export PYTHONPATH="${SYMBIFLOW}/bin/python:$BASE_DIR:$BASE_DIR/third_party/fasm:$BASE_DIR/third_party/edalize:$PYTHONPATH"
```

Comment:
This shouldn't be needed.

Use `pip install -e $BASE_DIR/third_party/fasm` or even better add it into the requirements.txt file

Reply:
Done, added also edalize in the requirements